### PR TITLE
Stork update ahead of workshop

### PR DESCRIPTION
**Pull request by Naktakala**
#24280 

## Reason
General update of stork that makes is work again.

@cticenhour

### CODE REVIEW

File: stork/src/base/StorkApp.C.app [hunk 1]
```diff
@@ -1,24 +1,32 @@
...
-  ModulesApp::registerAllObjects<StorkApp>(_factory, _action_factory, _syntax);
+  StorkApp::registerAll(_factory, _action_factory, _syntax);
 }
 
 StorkApp::~StorkApp() {}
 
+void 
+StorkApp::registerAll(Factory & f, ActionFactory & af, Syntax & /*s*/)
+{
+  Registry::registerObjectsTo(f, {"StorkApp"});
```
Comment (cticenhour) [inline stork/src/base/StorkApp.C.app:24]:
```suggestion
  ModulesApp::registerAllObjects<StorkApp>(_factory, _action_factory, _syntax);
  Registry::registerObjectsTo(f, {"StorkApp"});
```

We need to keep the ModulesApp registration in this method.

**Review by cticenhour — CHANGES_REQUESTED** [review]

File: stork/src/base/StorkApp.C.app [hunk 2]
```diff
@@ -1,24 +1,32 @@
 #include "StorkApp.h"
 #include "Moose.h"
 #include "AppFactory.h"
```
Comment (cticenhour) [inline stork/src/base/StorkApp.C.app:3]:
```suggestion
#include "AppFactory.h"
#include "ModulesApp.h"
```

Forgot this alongside my other comment.

Comment (Naktakala) [reply stork/src/base/StorkApp.C.app]: Done!

**Review by cticenhour — APPROVED** [review]

**Comment by loganharbour** [comment]
Unrelated failure.

**Pull request merged**